fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -75,7 +75,7 @@
         /** The amount of time taken loading a record store's {@code DataStoreInfo} header.*/
         LOAD_RECORD_STORE_INFO("load record store info"),
         /** The amount of time taken loading a record store's index meta-data. */
-        LOAD_RECORD_STORE_INDEX_META_DATA("load record store index meta-data"),
+        LOAD_RECORD_STORE_INDEX_META_DATA("load record store index meta-data", "ld_rs_idx_meta"),
```

Comment:
I understand that this isn't quite what this PR is about, but I'm not sure I love this log key shortening. But that's separate from my thoughts on the feature itself, which seems useful (if we can come up with a good standard for how we shorten these things).

Reply:
Yeah, as a suggestion, maybe the `StoreTimer` interface should have a section in the java docs with recommended abbreviations for the `logKey` as well as a rationale for why it exists.  We could them review and comment on the abbreviation set.  I think things like `rs` we can get used to seeing as long as we have a "dictionary" that defines it.  I think maybe we could also discuss things like if we should standardize on "read" (or "rd") instead of "load" (or "ld").  It'll drag the PR out a bit, but I think have a place to standardize on this stuff is worthwhile.

Reply:
Added some abbreviations and documentation about change frequency.

Reply:
Hm, I'm not sure what the right forum for this discussion would be, but one concern I have with the shorter names mapping is that it may make our metrics somewhat indecipherable (e.g., having to remember `ld_rec_val_byt_count` to `ld_idx_val_byt_count` to get total loaded bytes).

One other strategy might be to take this PR without the rosetta stone, and then pick a convention? Or maybe they aren't quite as inscrutable as I'm saying.

Reply:
I'm on board with deferring the naming convention for a second PR, but it should follow on this one relatively quickly.  I am sure that we won't reach consensus on brevity vs. legibility, so perhaps along with the draft of abbreviations, we should elect a benevolent dictator for which, after all opinions are in, simply makes the call.

Reply:
We should probably revert this line then if we're otherwise happy with how this is shaping up.